### DISCUSSION TITLE
Important

### DISCUSSION THREAD

**Discussion by Wiktor010** [mrdbourke/pytorch-deep-learning#676] [Q&A]
Hello, I would like to complete a project involving the creation of a vision system for classifying polymer details from the injection process. At the very beginning I had to write a script for the camera to process images. I have pre-processed these images. Now the images are fed to the input of CNN (SqueezeNet), which will classify a given detail into one of the defined classes. I am using a network model that is trained. I have a question for you if I have a model trained on ImageNet data. So is my task now just fine tuning and what next steps should I take? At the very end I would like to upload it to Raspberry Pi. classification should be at least 90%, and the processing time for one image should be less than 1500 ms.

**Comment by shisirkha** [comment]
Already we know that computer vision takes up a huge amount of memory. also doing it in a raspberry pi is quite slow.
also, I want to tell you that only the ImageNet data is not enough to properly recognize things in real-time.

if you want to recognize humans with your ML project you need to take photo of every angle a human can stand like each frame, and you will need different variants of human photos.

so, you need to be a photographer to do this and collect these data.
